platform/v1alpha1/types.go

```diff
@@ -0,0 +1,76 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	TypeSourced = "Sourced"
+	TypeApplied = "Applied"
+
+	ReasonSourceFailed     = "SourceFailed"
+	ReasonSourceSuccessful = "SourceSuccessful"
+	ReasonApplyFailed      = "ApplyFailed"
+	ReasonApplySuccessful  = "ApplySuccessful"
+)
+
+// PlatformOperatorSpec defines the desired state of PlatformOperator.
+type PlatformOperatorSpec struct {
+	// PackageName contains the desired OLM-based Operator package name
+	// that is defined in an existing CatalogSource resource in the cluster.
+	//
+	// This configured package will be managed with the cluster's lifecycle.
+	// +kubebuilder:validation:Required
+	// +required
```

Comment:
This tag doesn't exist and you have `kubebuilder:validation:Required` already, so please remove `+required` wherever it is present in this repo

Reply:
I'm happy to remove it here as the manifest didn't change when removing it, you're right. Do you want me to remove it across the entire repo in this PR or would that better served in a new issue?

Reply:
Just this file for now, I'll sort it across the repo separately